src/Illuminate/Container/ContextualBindingBuilder.php

```diff
@@ -66,4 +66,23 @@ public function give($implementation)
             $this->container->addContextualBinding($concrete, $this->needs, $implementation);
         }
     }
+
+    /**
+     * Define tagged services to be used as the implementation for the contextual binding.
+     *
+     * @param  string  $tag
+     * @return void
+     */
+    public function giveTagged($tag)
```

Comment:
I guess this also needs to be added to the ContextualBindingBuilder interface?

Reply:
Yeh, adding to an interface is a major breaking change.